sdk/storage/Azure.Storage.Blobs/samples/README.md

```diff
@@ -0,0 +1,20 @@
+---
+page_type: sample
+languages:
+- .NET
```

Comment:
dotnet, not .NET

Reply:
`dotnet` is a product slug, not [one of the devlangs](https://review.docs.microsoft.com/en-us/new-hope/information-architecture/metadata/taxonomies?branch=master#dev-lang).  We should use `csharp` instead.